### PR TITLE
KP-7617 Change function to also check shortname

### PR DESCRIPTION
**Pull request by kaisakui**
The changes query the corpus table for columns "metadata_urn" and "shortname".

### CODE REVIEW

File: roles/wordpress/files/kielipankki/popup.php [hunk 1]
```diff
@@ -346,53 +344,52 @@ function render_zotero($lang,$row, $key) {
...
+    $content .= localize($lang,"zotero_intro_text");
+    $content .='<pre class="light">'.render_zotero($lang,$row,$key).'</pre>';
     $content .='</div>';
-    $mysqli_connection->close();
-    return $content;
+    $content .='<p><a href="https://scholar.google.com/scholar?q='.urlencode(get_urn($row).' OR "'.get_title("fi", $row).'" OR "'.get_title("en", $row).'"').'" target="_parent">'.localize($lang,"search_scholar").'</a></p>';
+  } else {
+    $content .= localize($lang,"ref_heading")." <b>".$key."</b> ".localize($lang,"not_found").".<br>";
+    // $content .= localize($lang,"lb_notified");
+    $to = 'matthies@csc.fi'; //for now to test
```
Comment (Traubert) [inline roles/wordpress/files/kielipankki/popup.php:379]: So ultimately it should maybe be kielipankki@csc.fi?

File: roles/wordpress/files/kielipankki/popup.php [hunk 2]
```diff
@@ -346,53 +344,52 @@ function render_zotero($lang,$row, $key) {
...
+    $subject = 'Key error in reference instructions: '.$key;
+    $body = 'The reference instructions were called with an unknown key.<br>Please fix on the page where it came from.<p>';
+    $body .= 'Key: '.$key."<br>";
+    $body .= 'Language: '.$lang."<br>";
+    $body .= 'Referer: '. wp_get_referer();
+    $headers = array('Content-Type: text/html; charset=UTF-8');
+    // disable error mailing, referrer is not set properly in proxied machine
```
Comment (Traubert) [inline roles/wordpress/files/kielipankki/popup.php:386]: Should all this email stuff be commented out in that case?

File: README.md [hunk 3]
```diff
@@ -118,6 +118,15 @@ tag:
 
 This assumes that the pre-production server is otherwise ready and only needs a content sync.
 
+## Optional: Update Kielipankki Theme
+
+If you edit content in the Kielipankki theme (like popup.php or license.php) you can update it using the
+*kielipankki_theme* tag:
```
Comment (Traubert) [inline README.md:124]: I would maybe have called this `update_kielipankki_theme`, to indicate what the tag *does*, rather than what it *relates to*.

**Review by Traubert — APPROVED** [review]
It's PHP, so no need to be too fussy, but I added a couple of comments to maybe make it easier to understand.

File: roles/wordpress/files/kielipankki/popup.php (hunk 1 above)
Comment (mmatthiesencsc) [reply roles/wordpress/files/kielipankki/popup.php:379]: yes, but first we need to fix the problem that the HTTP_REFERRER information is lost in the proxying.
I finally made a bug report: https://jira.eduuni.fi/browse/KP-7707

File: roles/wordpress/files/kielipankki/popup.php (hunk 2 above)
Comment (mmatthiesencsc) [reply roles/wordpress/files/kielipankki/popup.php]: yes, makes it clearer.

File: README.md (hunk 3 above)
Comment (mmatthiesencsc) [reply README.md:124]: yes, changed.

**Pull request merged**